### DISCUSSION TITLE
Individual data storage

### DISCUSSION THREAD

**Discussion by santiago1229** [filecoin-project/community#683] [General]
Have been doing research and the plan sounds interesting for Filecoin. Are there opportunities at this point in time for an individuals to become the digital storage  hub for other users or is the development not quite there yet. 

Thanks

**Comment by wolfduan** [comment]
github

**Reply by santiago1229** [reply]
Pretty vague response. 

Can you link where you are talking about. 🙏

**Reply by santiago1229** [reply]
Moreover, how do I become a Miners, or storage node owners who can earn FIL?
